src/components/screens/IndexScreen/Hero.js

```diff
@@ -1,176 +1,361 @@
+        fill="url(#paint0_linear_195_11225)"
+      />
+      <defs>
+        <linearGradient
+          id="paint0_linear_195_11225"
+          x1="7"
+          y1="0"
+          x2="7"
+          y2="14"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stopColor="white" />
+          <stop offset="1" stopColor="white" stopOpacity="0" />
+        </linearGradient>
+      </defs>
+    </StarStyled>
+  );
+};
+
+export function Hero({ contributorCount, npmDownloads }) {
+  const [state, setState] = useState(false);
+  const [copiedValue, copyToClipboard] = useCopyToClipboard();
   const { latestVersion, urls = {} } = useSiteMetadata();
-  const { docs = {}, gitHub = {} } = urls;
+  const { docs = {}, gitHub = {}, npm } = urls;
+
+  let npmDownloadsFixed = parseInt((npmDownloads / 1000).toFixed(0), 10);
+  let npmDownloadsDisplay = `${npmDownloadsFixed}k`;
+  if (npmDownloadsFixed >= 1000) {
+    npmDownloadsFixed = (npmDownloadsFixed / 1000).toFixed(2);
+    npmDownloadsDisplay = `${npmDownloadsFixed}m`;
+  }
+
+  const onClick = () => {
+    copyToClipboard('npx storybook@latest init');
+    setState(true);
+    setTimeout(() => setState(false), 2000);
+  };
 
   return (
-    <Wrapper {...props}>
-      <PageLedeContainer>
-        <SectionLede
-          inverse
-          heading="Build UIs without the grunt work"
-          headingWrapper="h1"
-          copy="Storybook is a frontend workshop for building UI components and pages in isolation. Thousands of teams use it for UI development, testing, and documentation. It’s open source and free."
-          actions={
-            <>
-              <Button appearance="secondary" isLink href={docs} ButtonWrapper={GatsbyLinkWrapper}>
-                Get started
-              </Button>
-              <WithModal
-                startOpen={startOpen}
-                modal={Modal}
-                overlayStyles={{ backdropFilter: 'blur(10px)' }}
-              >
-                {({ onOpen }) => (
-                  <Button appearance="inverseOutline" onClick={onOpen}>
-                    <Icon icon="play" aria-hidden /> Watch video
-                  </Button>
-                )}
-              </WithModal>
-            </>
-          }
-          meta={
+    <Wrapper>
+      <ContentContainer>
+        <Title>Build UIs without the grunt work</Title>
+        <Description>
+          Storybook is a frontend workshop for building UI components and pages in isolation.
+          Thousands of teams use it for UI development, testing, and documentation. It&apos;s open
+          source and free.
+        </Description>
+        <Actions>
+          <Left>
+            <Buttons>
+              <ButtonSolid to={docs}>Get Started</ButtonSolid>
+              <ButtonInit onClick={onClick}>
+                npx storybook@latest init
+                <CopyIcon aria-hidden />
+                <AnimatePresence>
+                  {state && (
```

Comment:
I think you can replace your use of `useState` with this?

```suggestion
                  {copiedValue && (
```

Reply:
Never mind. Didn't notice your use of `setTimeout`.